### DISCUSSION TITLE
How do clients behave when the Remix server is updated?

### DISCUSSION THREAD

**Discussion by langri-sha** [remix-run/remix#6096] [Q&A] [answered]
Hi everyone! I was wondering if someone could point me to the docs or explain what happens in the case that, let's say, a client has loaded the Remix app on a certain route, and since then it has been updated with some breaking changes to the route actions and loaders?

**Comment by kiliman** [comment, marked as answer]
Here's another discussion about it
https://github.com/remix-run/remix/discussions/5654

**Reply by langri-sha** [reply]
Thanks a lot! I'll close this in favor of that discussion.
